docs/usage.rst

```diff
@@ -116,3 +116,65 @@ database handle (see `PDO::setAttribute`_).
 .. _DSN: https://en.wikipedia.org/wiki/Data_source_name
 .. _`PDO::setAttribute`: http://php.net/manual/en/pdo.setattribute.php
 .. _`CrateDB HTTP protocol`: https://crate.io/docs/reference/en/latest/protocols/http.html
+
+Examples
+========
+
+For creating a new connection please refer to `Connect to Crate`_.
+
+Insert
+------
+
+First prepare the statement:
+
+.. code-block:: php
+
+    $stmt = $connection->prepare('INSERT INTO test_table (id, array_type, object_type) VALUES(?, ?, ?)');
+
+Insert an integer value:
+
+.. code-block:: php
+
+    $stmt->bindValue(1, 1, PDO::PARAM_INT);
+
+Insert an array:
+
+.. code-block:: php
+
+    $stmt->bindValue(2, [1, 2], PDO::PARAM_ARRAY);
```

Comment:
`bindValue()` is same for `INSERT` and `UPDATE` so I would document it generally